pillar/fluentd/server_operations_qa.sls

```diff
@@ -1,38 +1,39 @@
-{% set app_name = "fluentd-aggregators" %}
-{% set fluentd_shared_key = salt.vault.read('secret-operations/operations-qa/fluentd_shared_key').data.value %}
-{% set heroku_http_token = salt.vault.read('secret-operations/operations-qa/heroku_http_token').data.value %}
-{% set mailgun_webhooks_token = salt.vault.read('secret-operations/operations-qa/mailgun_webhooks_token').data.value %}
-{% set odl_wildcard_cert = salt.vault.read('secret-operations/global/odl_wildcard_cert') %}
-{% import_yaml 'fluentd/fluentd_directories.yml' as fluentd_directories %}
-
-schedule:
-  refresh_{{ app_name }}_configs:
-    # Needed to ensure that S3 credentials remain valid
-    days: 5
-    function: state.sls
-    args:
-      - fluentd.config
+{% set ENVIRONMENT = salt.grains.get('environment') %}
+{% set mailgun_webhooks_token = salt.vault.read('secret-operations/{}/mailgun_webhooks_token'.format(ENVIRONMENT)).data.value %}
+{% set es_hosts = 'operations-elasticsearch.query.consul' %}
+{% set cert = salt.vault.cached_write('pki-intermediate-operations/issue/fluentd-server', common_name='{}'.format(es_hosts)) %}
```

Comment:
Why are we setting the elastic search hosts as the name for the clients certs?

Reply:
This would be the cert that the aggregators are using and the fluentd clients would be shipping their logs to that hostname. Unless I'm missing something, but believe that the name on the cert that the aggregators are using should match the hostname that the fluentd client are using to ship logs, no?

Reply:
Right, so shouldn't it be operations-fluentd.query.consul?

Reply:
That was a `duh` moment. Fixed